esmvaltool/recipes/recipe_collins13ipcc.yml

```diff
@@ -0,0 +1,2539 @@
+      - {dataset: GISS-E2-R-CC,     exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: HadGEM2-AO,       exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: HadGEM2-CC,       exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: IPSL-CM5A-LR,     exp: rcp85, start_year: 2006, end_year: 2300}
+      - {dataset: IPSL-CM5A-MR,     exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MIROC5,           exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MIROC-ESM,        exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MIROC-ESM-CHEM,   exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MPI-ESM-LR,       exp: rcp85, start_year: 2006, end_year: 2300}
+      - {dataset: MPI-ESM-MR,       exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MRI-CGCM3,        exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: MRI-ESM1,         exp: rcp85, start_year: 2006, end_year: 2100}
+      - {dataset: NorESM1-M,        exp: rcp85, start_year: 2006, end_year: 2100}
+
+    scripts:
+      ch12_ts_line_mean_spread_tas: &ts_settings
+        ## Required attributes:
+        script: ipcc_ar5/ch12_ts_line_mean_spread.ncl
+        #styleset: CMIP5
+        # Which scenarios shall be included in figure
+        scenarios: [rcp26, rcp45, rcp60, rcp85]
+        # Start years in time periods (e.g. start of historical period and rcps)
+        syears: [1850, 2006, 2101]
+        # end years in time periods (end year of historical runs and rcps)
+        eyears: [2005, 2100, 2300]
+        # start year of reference period (e.g. 1986)
+        begin_ref_year: 1986
+        # end year of reference period (e.g 2005)
+        end_ref_year: 2005
+        # labels to use in legend depending on scenarios
+        label: [Historical, RCP2.6, RCP4.5, RCP6.0, RCP8.5] 
+        ## Optional attributes:
+        # Specify title in plot  
+        title: "Global surface temperature change (~S~o~N~ C)"
+        # how many standard deviations to calculate the spread with? default is 1., ipcc tas used 1.64
+        spread: 1.64
+        # Specify y-axis title
+        yaxis: "(~S~o~N~ C)"
+        # Minimim value on y-axis, default calculated from data
+        ymin: -2
+        # Maximum value on y-axis          
+        ymax: 12
+        # Save number of model runs per period and scenario in netcdf to print in plot? (True, False)
+        model_nr: True
+        # Choose file to create colormap from or specify ncl colormap
+        colormap: ./diag_scripts/shared/plot/rgb/ipcc_color_tseries.rgb
+        # Minimum and maximum latitudes and longitudes if not global
+        #ts_minlat: 43
+        #ts_maxlat: 54.5
+        #ts_minlon: -12
+        #ts_maxlon: 35
+      ch12_plot_ts_line_mean_spread_tas:
+        <<: *ts_settings
+        script: ipcc_ar5/ch12_plot_ts_line_mean_spread.ncl
+        ancestors: ['tas', 'ts_line_tas/ch12_ts_line_mean_spread_tas']
```

Comment:
Replace by
```yaml
        ancestors: ['tas', 'ch12_ts_line_mean_spread_tas']
```
for easier reading. There is no need to prefix the diagnostic name to the ancestors when using ancestors from the same diagnostic. You should only need to prefix the diagnostic name when using variables or diagnostic scripts from other diagnostics.